projects/components/src/button/button.component.scss

```diff
@@ -123,6 +135,10 @@
     @include button-style(inherit, inherit, $blue-5, $blue-4);
   }
 
+  &.primary-reversed {
```

Comment:
This line needs to change also.

Reply:
Removed here https://github.com/hypertrace/hypertrace-ui/pull/1123